buildroot-external/patches/occu/0021-WebUI-Remove-Logo/occu/WebUI/www/rega/login.htm

```diff
@@ -69,7 +69,7 @@
         <td id="navbar">
           <table cellpadding="0" cellspacing="0">
             <tr>
-              <td style="text-align: center;"><div style="width: 180px;"><img src="/ise/img/homematic_logo_small.png" alt="HomeMatic Logo" /></div></td>
+              <td style="text-align: center;"><div style="width: 180px;"><img src="/ise/img/rm-logo_small.png" alt="RaspberryMatic Logo" /></div></td>
```

Comment:
These and the following changes to `login.htm` and `logout.htm` files in the other WebUI patchsets seem to be unnecessary. A usual better approach would be to always use the original files from the occu github repository as the base files.

Reply:
I have reverted that and squashed all commits into a new one. Please check